pkgs/development/tools/dt-schema/default.nix

```
@@ -7,23 +7,31 @@
 , rfc3987
 , setuptools
 , setuptools-scm
+, libfdt
 }:
 
 buildPythonPackage rec {
   pname = "dtschema";
-  version = "2022.1";
+  version = "2022.12";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-G5KzuaMbbkuLK+cNvzBld1UwvExS6ZGVW2e+GXQRFMU=";
+    sha256 = "sha256-9TLUM6kV5QfUtCa371emcw0jwJOLaC+0hm+d+oCljsY=";
   };
 
+  patches = [
+    # Change name of pylibfdt to libfdt
+    ./fix_libfdt_name.patch
+  ];
+
   nativeBuildInputs = [ setuptools-scm git ];
```

```suggestion
  nativeBuildInputs = [ setuptools-scm ];
```
tested locally and not required